src/genezio.ts

```diff
@@ -229,12 +228,18 @@ program
     )
     .option("--language <language>", "Language of the SDK.", "ts")
     .option(
-        "-s, --source <source>",
+        "--source <source>",
+        "Where the SDK should be generated from. Possible values: local, remote.",
+        "./",
+    )
+    .option(
+        "-c, --config <config>",
         "Path to the genezio.yaml file on your disk. Used for loading project details from a genezio.yaml file, instead of command argumments like --name",
         "./",
     )
     .option("-p, --path <path>", "Path to the directory where the SDK will be generated.", "./sdk")
     .option("--stage <stage>", "Stage of the project.", "prod")
+    .option("--url <url>", "The url of the server.")
```

Comment:
```suggestion
    .requiredOption("--url <url>", "The url of the server.")
```

This option has no default, but it was defined as it can't be undefined in the `GenezioSdkOptions` type. Please make it required.